### DISCUSSION TITLE
Three Rivers District Council Bin Collection feature

### DISCUSSION THREAD

**Discussion by rajp1503** [mampfes/hacs_waste_collection_schedule#1638] [General]
Is it possible to get the scanning of the Three Rivers District Council bin collection info? At present, we have to sign up for a email which is sent out by them the day prior to collection. It is also possible to look on their website at https://my.threerivers.gov.uk too.

**Comment by 5ila5** [comment]
please open a [Source request](https://github.com/mampfes/hacs_waste_collection_schedule/issues/new?assignees=&labels=source+request&projects=&template=SOURCE-REQUEST.yml&title=%5BSource+Request%5D%3A+)
